.eslintrc.json

```diff
@@ -126,6 +127,9 @@
     "@typescript-eslint/no-namespace": "off",
     "@typescript-eslint/prefer-namespace-keyword": ["off"],
 
+    // Show errors for deprecated code
+    "deprecation/deprecation": "error",
```

Comment:
Another option would be to leave these as warnings instead of errors, which would allow us to still take updates that deprecate things we use, while giving us the opportunity to learn about them locally with some tooling support.

Reply:
Ehhhh I kind of feel like warnings just lead to broken windows

Reply:
Let's try it and see if it becomes annoying. I'm open to reconsidering down the road.